calcom/cal.com
# fix: Adding yearly title to preview page

## What does this PR do?

I have configured this issue, and the problem was that when checking the condition for frequency, it was ignoring 0 and only accepting 1 and 2. Here, frequency 0 represents 'year', 1 represents 'month', and 2 represents 'week'.

I made no changes in the initial commit when I forked the project. I automatically pushed after installing the packages using Yarn.

- Fixes #16432 (GitHub issue number)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1. Go to event-types page.
2. Select any one event.
3. Enable recurring , ( for year )
4. Now preview your page , you will see the desire result.

![Screenshot 2024-08-31 154658](https://github.com/user-attachments/assets/344c833d-6f1a-4a63-a903-d9c7eef43a7d)
